Park-Swift/bin/install

```diff
@@ -6,4 +6,8 @@ npm install @react-navigation/material-top-tabs react-native-tab-view
 
 nvm install node --reinstall-packages-from=current
 
-npx expo install react-native-pager-view
\ No newline at end of file
+npx expo install react-native-pager-view
+
+npm install react-native-keyboard-aware-scroll-view
```

Comment:
Great job updating the install scripts! This made it super easy for me to test the app and run it locally. Perfect to review it